pkg/resourceinterpreter/default/native/prune/prune.go

```diff
@@ -98,9 +98,9 @@ func removeGenerateSelectorOfJob(workload *unstructured.Unstructured) error {
 		return err
 	}
 	if exist {
-		if util.GetLabelValue(matchLabels, "controller-uid") != "" {
-			delete(matchLabels, "controller-uid")
-		}
+		util.DeleteLabelIfExist(matchLabels, "controller-uid")
```

Comment:
Why not  just use `delete(matchLabels, "controller-uid")`?

Reply:
You're right, i just want to keep the code code style but ignoring this step is unnecessary.